.github/workflows/ci.yaml

```diff
@@ -0,0 +1,57 @@
+name: godot-go CI
+
+on:
+  pull_request:
+    paths:
+      - "**/*.go"
+      - "**/*.c"
+      - "**/*.h"
+      - ".github/workflows/ci.yaml"
+
+  # push:
+  #   paths:
+  #     - "**/*.go"
+  #     - "**/*.c"
+  #     - "**/*.h"
+  #     - ".github/workflows/ci.yaml"
+
+jobs:
+  run_test:
+    name: "run tests"
+    strategy:
+      fail-fast: true
+      matrix:
+        go_version:
+          - 1.19.x
+        targets:
+          - linux/amd64
+          # - darwin/amd64
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go_version }}
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install Godot 4.0 beta1 server
+        run: |
+          wget https://downloads.tuxfamily.org/godotengine/4.0/beta1/Godot_v4.0-beta1_linux.x86_64.zip
```

Comment:
looks super nice, I didn't know beta server zips were available :+1:

Reply:
i've decided to keep the godot_headers in sync with the latest beta to make things easier for everyone out there since it's critical that godot-go and your godot binaries are running on the same godot_headers.